dbms/src/Interpreters/Join.cpp

```diff
@@ -1892,7 +1889,7 @@ void Join::joinBlockImplCrossInternal(Block & block, ConstNullMapPtr null_map [[
     for (size_t start = 0; start <= rows_left; start += left_rows_per_iter)
     {
         size_t end = std::min(start + left_rows_per_iter, rows_left);
-        MutableColumns dst_columns(num_existing_columns + num_columns_to_add);
+        MutableColumns dst_columns(block.columns());
         for (size_t i = 0; i < block.columns(); i++)
```

Comment:
```suggestion
        for (size_t i = 0; i < block.columns(); ++i)
```

Reply:
from https://clickhouse.com/docs/en/development/style/

Reply:
OK, there are other twelve `i++` in `Join.cpp`.🤣